packages/react-composites/src/composites/CallComposite/pages/UnsupportedBrowser.tsx

```diff
@@ -16,7 +14,6 @@ import { useLocale } from '../../localization';
  */
```

Comment:
Should this be @private instead of @internal (@internal is for stuff you want to use across packlets, but not export in the public API. @private is for stuff restricted to a single packlet)

Reply:
+1